src/Public/OpenApi.ps1

```diff
@@ -1985,25 +2002,31 @@ function Enable-PodeOAViewer {
             }
             $PodeContext.Server.OpenAPI.Definitions[$DefinitionTag].hiddenComponents.bookmarks.route += $route
         }
-
     }
     else {
         if ($Type -ieq 'RapiPdf' -and (Test-PodeOAVersion -Version 3.1 -DefinitionTag $DefinitionTag)) {
             # The Document tool RapidPdf doesn't support OpenAPI 3.1
             throw ($PodeLocale.rapidPdfDoesNotSupportOpenApi31ExceptionMessage)
         }
         # set a default path
-        $Path = Protect-PodeValue -Value $Path -Default "/$($Type.ToLowerInvariant())"
+        $Path = Protect-PodeValue -Value $Path -Default "/$($Type.ToLowerInvariant())/"
```

Comment:
`/$($Type.ToLowerInvariant())/` here needs to be reverted to the original `/$($Type.ToLowerInvariant())`